src/cdk/a11y/live-announcer/live-announcer.ts

```diff
@@ -94,8 +99,17 @@ export class LiveAnnouncer implements OnDestroy {
     this.clear();
     clearTimeout(this._previousTimeout);
 
+    if (!politeness) {
+      politeness =
+          (defaultOptions && defaultOptions.politeness) ? defaultOptions.politeness : 'polite';
+    }
```

Comment:
Nit: any reason why this is not consistent to how the `duration` is handled? e.g.

```ts
if (!politeness && defaultOptions) {
  politeness = defaultOptions.politeness;
}

if (duration == null && defaultOptions) {
  duration = defaultOptions.duration;
}

// Politeness fallback inline (how it was before).

Reply:
It's because we want to default the politeness even if there aren't any `defaultOptions`. The `duration` checks against `null`, because it's a `number` so it could be 0 in theory.

Reply:
Duration makes sense. but do I miss something? We will still have a fallback in _all cases_ while not needing the ternary and being consistent? I might miss something.

```ts
if (!politeness && defaultOptions) {
  politeness = defaultOptions.politeness;
}

...

// Politeness fallback inline (how it was before).
this._liveElement.setAttribute('aria-live', politeness! || 'polite'); <---
```

Reply:
It not really about consistency. I was trying to avoid having the values for `politeness` in two places: both in the `if` and the `||` at the end.

Reply:
Sure. I don't feel like it's too important.